lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1632,6 +1659,12 @@ bool TFGraphLowering::copyGraphFunctions(const std::vector<char> &graphDefProto,
 /// Lower a graph_op into the TensorFlow op node.
 ///
 GLStatus TFGraphLowering::visitGraphOperationInst(GraphOperationInst *inst) {
+  // If this is the magic tf_tensor_to_i1 builtin, then we completely ignore it.
+  // the only user of it are things that take conditional branches, and they
+  // handle it directly.
+  if (inst->getName().str() == "tf_tensor_to_i1")
+    return GLStatus::Success;
```

Comment:
Shouldn't this do:
      auto opValue = getOperandValue(inst->getOperand(0));
      if (!opValue.oper) return GLStatus::Error;
     addValueMapping({inst->getResult(0), 0}, {result, 0});

before returning?